lib/pdf/core/page.rb

```diff
@@ -167,6 +173,9 @@ def init_new_page(options)
         @dictionary = document.ref(:Type        => :Page,
                                    :Parent      => document.state.store.pages,
                                    :MediaBox    => dimensions,
+                                   :CropBox     => dimensions,
+                                   :BleedBox    => dimensions,
```

Comment:
`CropBox` is optional and by default is the same as `MediaBox`. `BleedBox` is also optional and by default is the same as `CropBox` (and thus if that's left out — same as `MediaBox`).

Are these things required for PDF/X?